src/Command/CommandHelper.php

```diff
@@ -96,7 +96,7 @@ public static function begin(
 		if ($allowXdebug && !XdebugHandler::isXdebugActive()) {
 			$errorOutput->getStyle()->note('You are running with "--xdebug" enabled, but the Xdebug PHP extension is not active. The process will not halt at breakpoints.');
 		} elseif (!$allowXdebug && XdebugHandler::isXdebugActive()) {
-			$errorOutput->getStyle()->note('The Xdebug PHP extension is active, but "--xdebug" is not used. This may slow down performance and the process will not halt at breakpoints.');
+			$errorOutput->getStyle()->note('The Xdebug PHP extension is active, but "--xdebug" is not used. This may slow down performance and the process will not halt at breakpoints. Run with XDEBUG_MODE=off to disable Xdebug and remove this message.');
```

Comment:
I'd argue that this message isn't still clear enough. What does "Run with XDEBUG_MODE=off" mean?

Reply:
I agree that it's not totally clear, but phpstan is a *developer* tool, and to *developers* this is at least a firm push in the right direction. A better explanation would take at least 4 lines, likely more, and at that point I think you're crossing the line it should be explained *thoroughly* at https://phpstan.org/user-guide/command-line-usage, and the warning itself should just be as it is now.

If the verbose explanation is in the docs people who need it can and will google "phpstan xdebug_mode=off" and find it right away.